src/api/layers/ExternalWMSLayer.class.js

```diff
@@ -47,6 +47,9 @@ export default class ExternalWMSLayer extends ExternalLayer {
      *   Default is `true`
      * @param {String} externalWmsData.baseUrl Base URL to build WMS requests (no endpoint / URL
      *   param defined)
+     * @param {ExternalWMSLayer[]} externalLayerData.layers Description of the layers being part of
+     *   this group (they will all be displayed at the same time, in contrast to an aggregate
```

Comment:
maybe rename this comment from `this group` to `this WMS layer` or something like that